packages/@webex/plugin-meetings/src/rtcMetrics/index.ts

```diff
@@ -0,0 +1,97 @@
+import uuid from 'uuid';
+
+const appId = 'FFB51ED5-4319-4C55-8303-B1F2FCCDE231';
```

Comment:
i would move all the constants to a constants file in the rtcMetrics , can be done in next PR too

Reply:
Added constants file.